### PR TITLE
small fixes in Dockerfile

### PR DESCRIPTION
**Pull request by luclaurent**
Small fixes on typo and reduction of the number of layer. This update does not require a version's increment.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -12,37 +13,39 @@ RUN apt update && \
...
+# switch to "user"
 USER user
 
-#working dir
+# working dir
 WORKDIR /tmp/work
 
-#declare directory using environement 
+# declare useful directories using environement 
 ENV MONITOR_DIR=/tmp/work
```
Comment (nennigb) [inline docker/Dockerfile:49]: Perhaps we can use the variable name everywhere for clarity.

File: docker/Dockerfile [hunk 2]
```diff
@@ -12,37 +13,39 @@ RUN apt update && \
...
 RUN git clone https://github.com/nennigb/amc2moodle.git -b master
 WORKDIR /tmp/amc2moodle
-RUN pip3 install .
+RUN pip3 install -U pip && \
+    pip3 install .
 
 # check if amc2moodle and moodle2amc work
 WORKDIR /
 RUN python -m amc2moodle.amc2moodle.test && \
     python -m amc2moodle.utils.test
```
Comment (nennigb) [inline docker/Dockerfile:28]: The moodle2amc tests are not present here. Please find the CI lines
```
    - name: Test moodle2amc
      run: |
        # add automultiplechoice.sty local copy to LaTeX PATH (for this step)
        export TEXINPUTS=.:./amc2moodle/moodle2amc/test:$TEXINPUTS
        python -m amc2moodle.moodle2amc.test
```

**Review by nennigb — APPROVED** [review]
Hi,
Thank for the cleaning. I list two ways of (small) improvements (see the file).
Have you test the container building and execution at least locally ?

File: docker/Dockerfile (hunk 1 above)
Comment (luclaurent) [reply docker/Dockerfile]: Done!

File: docker/Dockerfile (hunk 2 above)
Comment (luclaurent) [reply docker/Dockerfile]: I have added it.

**Comment by luclaurent** [comment]
I have tested the building locally.

**Comment by nennigb** [comment]
You can merge it when you are ready ;-)

**Pull request merged**